packages/siop-oid4vp/CHANGELOG.md

```diff
@@ -0,0 +1,244 @@
+# Release Notes
+
+The DID Auth SIOP typescript library is still in an beta state at this point. Please note that the interfaces might
+still change a bit as the software still is in active development.
+
+## 0.6.5
```

Comment:
@nklomp how do you want to deal with versioning? Do we want to adjust it based on the current oid4vci version and then release them at the same time?